frontend/src/material/Box.tsx

```diff
@@ -33,7 +33,7 @@ const Box = React.forwardRef(
                 {...otherProps}
                 ref={ref}
                 className={classNames({
-                    [props.className]: true,
+                    [props.className]: !!props.className,
```

Comment:
If it is only for boolean conversion, a convenient way could be Boolean(props.className) ? Not sure if it would work

Reply:
Much more sexy indeed!